src/parser/parser.spec.ts

```diff
@@ -340,7 +336,6 @@ describe('fabric.Parser', () => {
   it('parseTransformAttribute', () => {
     let parsedValue;
 
-    expect(parseTransformAttribute).toBeDefined();
```

Comment:
these make 0 sense to assert, since tests would break on compile if they wouldn't exist
it made sense back then with qunit but with vitest is redundant